bin/knc.c

```diff
@@ -999,40 +1007,41 @@ write_network_buffer(work_t *work)
 	return len;
 }
 
-#define MAX(a,b)	(((a) > (b)) ? (a) : (b))
-
 int
 move_data(work_t *work)
 {
+	struct pollfd pfds[4];
+	nfds_t nfds;
+	struct pollfd * const pfd_net = pfds+0;
+	struct pollfd * const pfd_in  = pfds+1;
+	struct pollfd * const pfd_out = pfds+2;
+	struct pollfd * const pfd_err = pfds+3;
```

Comment:
Why the `const`?

Reply:
It is an indication to the reader that those values are not going to change in the struct pollfd list pfds[] in how this routine uses that list.  Also, it is a (minor) compiler hint to the optimizer.